doc/security/specs/identities_and_root_keys/_index.md

```diff
@@ -0,0 +1,821 @@
+   retain the Creator Identity across validated updates of the ROM_EXT.
+   The implementation may opt to use the software binding interface
+   described in later sections to fulfill this property.
+    </td>
+  </tr>
+  <tr>
+    <td>Device Identifier</td>
+    <td>OTP</td>
+    <td>
+Provisioned at manufacturing time. Readable from software and JTAG interface.
+    </td>
+  </tr>
+  <tr>
+    <td>Hardware Revision Secret</td>
+    <td>Gates</td>
+    <td>
+Encoded in gates. Provisioned by Silicon Creator before tapeout. Hidden from
+software.
+    </td>
+  </tr>
+  <tr>
+    <td>Health State Measurement</td>
+    <td>Register (ROM stage)</td>
+    <td>
+Comprises the following measurements:
+
+* Device life cycle state.
+* Debug mode state.
+* ROM Hash.
+
+Some values are read from the device lifecycle controller. The device life cycle
+state should be consumed by the ROM stage.
+
+The debug mode shall be used as well if there are multiple debug configurations
+supported by a single lifecycle state.
+    </td>
+  </tr>
+</table>
+
+The CreatorRootKey can be used to generate the Creator Identity key and the
+Owner Intermediate Key described in the following sections.
+
+## Creator Identity
+
+The Creator Identity is an asymmetric key derived from the CreatorRootKey. It is
+used as a cryptographic identifier bound to the device and the Silicon Creator.
+It is used to attest to the authenticity of the physical device and the ROM and
+ROM\_EXT configuration.
+
+The Creator Identity is generated as follows:
+
+```
+CreatorIdentitySeed =
+  KM_DERIVE(CreatorRootKey, IdentityDiversificationConstant)
+
+// ASYM_KDF is a KDF function compliant to the Asymmetric Key
+// requirements defined in the Attestation specification document.
+CreatorIdentity_Private = ASYM_KDF(CreatorIdentitySeed)
+CLEAR_BEFORE_NEXT_BOOT_STAGE(CreatorIdentitySeed, CreatorIdentity_Private)
+```
+
+<table>
+  <tr>
+    <td><strong>Name</strong></td>
+    <td><strong>Encoding</strong></td>
+    <td><strong>Description</strong>
+    </td>
+  </tr>
+  <tr>
+    <td>Identity Diversification Constant</td>
+    <td>Gates</td>
+    <td>
+A constant defined in gates. Used to derive the CreatorIdentitySeed from the
+CreatorRootKey.
+
+Hidden from software.
+    </td>
+  </tr>
+  <tr>
+    <td>Creator Identity Seed</td>
+    <td>SW Register Output</td>
+    <td>Seed used to generate the CreatorIdentity asymmetric key.</td>
+  </tr>
+</table>
+
+The CreatorIdentitySeed and the private portion of the Creator Identity shall be
+cleared before the ROM Extension hands over execution to the Silicon Owner first
+boot stage.
+
+## Owner Intermediate Key {#owner-intermediate-key}
+
+The Owner Intermediate Key is used as a root component of the Silicon Owner key
+hierarchy. It is used to establish a cryptographic link to the root secrets
+provisioned at manufacturing time.
+
+**Visibility**
+
+The OwnerIntermediateKey shall be kept hidden from software to mitigate owner
```

Comment:
Similarly you switch between Owner Intermediate Key and OwnerIntermediateKey. Personally I think it would be better to have defined objects in `backtick` representation with their CamelCase names. But as long as you're consistent in the document.

Reply:
Updated for improved consistency. All variables are defined in CamelCase, except for Owner Identity and Creator Identity. `backtick` used in most places except inside tables due to formatting issues.

Reply:
Updated commit coming up :)